app/assets/stylesheets/ama_layout/layout_components/error-messages.scss

```diff
@@ -1,10 +0,0 @@
-.notice_notification{
```

Comment:
@kathleenAMA you'll have to re-add when you merge `master` into `registries`:

`git add app/assets/stylesheets/ama_layout/layout_components/error-messages.scss`